src/Service/Sqs/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## NOT RELEASED
 
+### Added
+
+- AWS api-change: This release enables customers to call SQS using AWS JSON-1.0 protocol.
+- AWS api-change: Amazon SQS adds a new queue attribute, SqsManagedSseEnabled, which enables server-side queue encryption using SQS owned encryption keys.
```

Comment:
```suggestion
- AWS api-change: This release enables customers to call SQS using AWS JSON-1.0 protocol.
```


ouch.. this is a huge change :exploding_head: . No "fake sqs" image is available yet

Reply:
this have been reverted...